tests/phpunit/tests/link/getEditCommentLink.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * @group link
+ * @group comment
+ * @covers ::get_next_comments_link
+ */
+class Tests_Link_GetEditCommentLink extends WP_UnitTestCase {
+
+	public static $comment_ids;
+	public static $user_ids;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
```

Comment:
You'll need to add a corresponding `wpTearDownAfterClass()` method that deletes the new comment and the new users again (after all tests in the class have been run).